tools/render/render_trace.cc

```diff
@@ -25,8 +42,20 @@ int main(int argc, char* argv[]) {
   CHECK_GE(argc, 2) << "Specify file path";
   auto trace = absl::make_unique<quic_trace::Trace>();
   {
-    std::ifstream f(argv[1]);
-    trace->ParseFromIstream(&f);
+    std::string filename(argv[1]);
+    std::ifstream f(filename);
+    if (GuessInputFileFormat(filename) == INPUT_QTR) {
```

Comment:
Could you turn this into a switch statement, so that if we add more formats, this would hit a compiler warning?

Reply:
Definitely. Fixed in d433e960a5eb008d38cb7110e7d708abf4e426ec.